po/pt.po

```diff
@@ -648,7 +1017,7 @@ msgstr "A procurar atualizações no AUR"
 
 #: src/upgrade.rs:148
 msgid "Looking for devel upgrades"
-msgstr "A procurar atualizações de pacotes git"
+msgstr "À procura de atualizações de desenvolvimento"
```

Comment:
I think "A procurar" is better than "à procura", most portuguese translations have the verb in infinitive form. In the pacman translation it is also "A procurar"

Reply:
Correction done